mm2src/mm2_test_helpers/src/for_tests.rs

```diff
@@ -845,7 +845,10 @@ fn eth_conf(coin: &str) -> Json {
         "chain_id": 1337,
```

Comment:
redundant chain_id?
also found in other places in this file.

Reply:
Not redundant as this config can be used for either v1 or v2 tests.

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/commit/2b44429a3522596d12128316b85270b9a067ce2a